srcpkgs/m1n1/files/kernel.d/m1n1.post-install

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# Kernel hook for m1n1,
+# taken from update-m1n1 and efibootmgr.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+
+set -e
+
+PKGNAME="$1"
+VERSION="$2"
+
+[ -e "${ROOTDIR}/etc/default/m1n1-kernel-hook" ] && . "${ROOTDIR}/etc/default/m1n1-kernel-hook"
+[ -d "${BOOT}" ] || return 0
+
+m1n1config=/run/m1n1.conf
+>"$m1n1config"
```

Comment:
what is this doing?

Reply:
it creates an empty file with that name.
(this is taken from asahi-scripts update-m1n1)

Reply:
`: >path` is a little clearer on intent and doesn't look like an incomplete line

Reply:
done.